test/pylib/scylla_cluster.py

```diff
@@ -1249,6 +1278,21 @@ async def _after_test(self, _request) -> str:
             self.current_test_case_full_name = ''
         self.is_after_test_ok = True
         cluster_str = str(self.cluster)
+
+        # cancel all other tasks in ScyllaClusterManager
+        self.logger.info(self.repr_tasks_history())
+        for key, value in self.tasks_history.items():
+            if "after-test" not in value.path_qs:
+                key.cancel(msg="test finished")
```

Comment:
Shouldn't we wait for tasks to finish instead of cancelling them? (Not sure what cancel does though)

Reply:
it can be too long. For example in reproducer  https://github.com/scylladb/scylladb/issues/16651#issue-2066160420

I faced an issue that waiting node_start took TOPOLOGY_TIMEOUT = 1000 sec. I think it is a quite big timeout especially if we are in _after_test. meaning the test has already been finished